examples/resources/awscc_sso_assignment/use_new_permission_set.tf

```diff
@@ -0,0 +1,59 @@
+// Note: Currently there is no data source for fetching the IAM Identity Center (formerly AWS SSO)
+//instance arn in the awscc provider so we must use both the aws and awscc providers.
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+    awscc = {
+      source = "hashicorp/awscc"
+    }
+  }
+}
+
```

Comment:
for brevity , you can remove this portion

Reply:
updated and pushed changes 👍🏾